completion/tests/test_models.py

```diff
@@ -142,3 +142,69 @@ def test_submit_batch_completion_without_waffle(self):
         with self.assertRaises(RuntimeError):
             blocks = [(self.block_key, 1.0)]
             models.BlockCompletion.objects.submit_batch_completion(self.user, self.course_key, blocks)
+
+
+class BatchCompletionMethodTests(CompletionSetUpMixin, TestCase):
+    """
+    Test model methods:
+        latest completion per course, and all completions per course
+    """
+    COMPLETION_SWITCH_ENABLED = True
+
+    def setUp(self):
+        super(BatchCompletionMethodTests, self).setUp()
+        self.set_up_completion()
+        self.user = UserFactory.create()
```

Comment:
This comes for free from the `CompletionSetUpMixin` class.

Reply:
Although, maybe since you also have `self.other_user`, it's not a bad idea to just spend the extra line and re-define it (like you're doing).  So I guess just leave it.

Reply:
I'll change it to `user_one` and `user_two`. Compromise!